source/js/classes/Timer.js

```diff
@@ -34,36 +34,47 @@ class Timer {
    */
   startTimer() {
     return new Promise((resolve) => {
-      // Count down immediately
-      // This is where the timer callbacks a function every second
       const resolveMessage = 'Timer Finished';
-      if (this.callbackEverySecond !== null) {
-        this.callbackEverySecond(this.minutes, this.seconds);
-      }
-      if (this.seconds === 0 && this.minutes !== 0) {
-        this.minutes -= 1;
-        this.seconds = 60;
-      } else if (this.seconds === 0 && this.minutes === 0) {
+      if (this.seconds === 0 && this.minutes === 0) {
```

Comment:
I still got the negative numbers after the fix, do we need to change this to <= also?
Is there anywhere else that we need to change?